src/github-issues-prs.ts

```diff
@@ -48,6 +48,7 @@ export class GitHubIssuesPrsProvider implements TreeDataProvider<TreeItem> {
 	private children: Promise<TreeItem[]> | undefined;
 
 	private username: string | undefined;
+	private host: string | undefined;
```

Comment:
getGitHubRemotes() has two references to "github.com". I guess that needs to be updated too. Do you have access to a GitHub Enterprise install?

Reply:
Thanks, I'll fix.
But, I don't have GHE. How should I test for GHE environment?

Reply:
OK fixed!